docs/reference/python-sdk.rst

```diff
@@ -4,17 +4,39 @@
  Python SDK
 ############
 
-You can interact with a Determined cluster with the Python SDK.
+You can interact with a Determined cluster using the Python SDK. The Determined Python SDK, a part
+of the broader Determined Python library, is designed to perform tasks such as:
+
+-  Creating and organizing experiments
+-  Downloading model checkpoints, and adding them to the model registry
+-  Retrieving trial metrics
 
 The client module exposes many of the same capabilities as the det CLI tool directly to Python code
 with an object-oriented interface.
 
-*****************************
- Experiment Workflow Example
-*****************************
+*****************************************
+ Example: Run and Administer Experiments
+*****************************************
+
+This `demo <https://github.com/determined-ai/determined-examples/tree/main/blog/python_sdk_demo>`__
```

Comment:
question: is it a good idea to link "live" docs to "unmaintained examples"?

Maybe it is, I'm not sure.  Or maybe it's not, but the answer is to promote the sdk demo to the determined repo rather than to delete the reference from the docs.

Reply:
for clarification, we would want to promote the sdk demo

from here:

This repository contains a variety of Determined examples that are **not** actively maintained by the Determined team.

https://github.com/determined-ai/determined-examples/tree/main

to here:

This directory only contains examples that **are routinely tested** on our CI platform.

https://github.com/determined-ai/determined/tree/main/examples

?